Error starting orgbook-web (frontend) container when deployed to OpenShift

After merging https://github.com/bcgov/orgbook-bc-client/pull/185 the updated code was built the deployed into OpenShift.  The resulting container is failing to start with the following error:

```
panic: Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.21 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.21 set. Notably, if go1.21 adds a moving garbage collector, this program is unsafe to use.

goroutine 1 [running]:
go4.org/unsafe/assume-no-moving-gc.init.0()
	go4.org/unsafe/assume-no-moving-gc@v0.0.0-20220617031537-928513b29760/untested.go:25 +0x1da
```

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -13,9 +13,9 @@ RUN npm run build
 # caddy build stage
 FROM ${caddy_builder_image} as caddy-build-stage
 
-# Install ratelimit/v0.2.0
+# Install ratelimit/v0.3.0
 RUN xcaddy build \
-    --with github.com/RussellLuo/caddy-ext/ratelimit@3497b19617f19deca8a4a625a93d2b774bb0dd29
+    --with github.com/RussellLuo/caddy-ext/ratelimit@ab1e18cfe426012af351a68463937ae2e934a2a1
 
 # production stage
 FROM ${runtime_image} as production-stage

